lib/ConfigManager/ConfigManager.js

```diff
@@ -60,17 +60,20 @@ class ConfigManager extends React.Component {
   }
 
   onSave(data) {
-    const { resources, mutator, moduleName, configName, calloutMessage, onBeforeSave, onAfterSave } = this.props;
+    const { resources, mutator, moduleName, configName, scope, calloutMessage, onBeforeSave, onAfterSave } = this.props;
     const value = (onBeforeSave) ? onBeforeSave(data) : data[configName];
-    const setting = Object.assign({}, resources.settings.records[0], {
-      module: moduleName,
-      configName,
-      value,
-    });
+    // eslint-disable-next-line prefer-object-spread
```

Comment:
Ah, yes, that. The ESLint disabler wasn't in the original version of this component, and it wasn't (and still isn't) needed because [the rule is already disabled in this repository](https://github.com/folio-org/stripes-smart-components/blob/619df71dacfdd482d4785886bab7023542087f91/.eslintrc#L9). I added it when I was developing the changes inside my app, which is less lax in its linting. I'll remove it.